### PR TITLE
refactor: add contribution file and copyright info

### LINKED ISSUE #483: Suggestion: set reasonable and necessary contribution guidelines
To maintain the code quality and overall maintainability of the Ozhera project, we need to establish clear guidelines for code contributions. 
A solid coding style and consistent formatting checks are vital for any open-source initiative. 
I suggest we create the following documentation to assist new contributors: 
1. A thorough coding style and formatting specification, including codestyle and checkstyle; 
2. A clear and accurate process for code updates and submitting PRs; 
3. A detailed procedure for reporting vulnerabilities and bugs; 
4. Comprehensive guidelines for unit testing; 
5. Issues should feature tags like `good first issue` to foster greater engagement. 

Moreover, I believe we should work on enhancing community activity. Thus, we should also consider developing a set of rules for incentivizing contributor transitions (from contributor to committer, ultimately to PPMC member) in line with ASF standards.

### PR DESCRIPTION
**Pull request by psxjoy**
resolve #483 
add first version of contribution guidelines.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md
Comment (gaoxh) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md:1]: Please rename the project name and link in the document

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md
Comment (gaoxh) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:1]: Please rename the project name and link in the document

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md
Comment (psxjoy) [reply .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md:1]: Thank you for your code review. This is temporary process data（WIP）, I will make relevant updates (including deleting or modifying documents) when the official PR is submitted.
------
谢谢code review。这是暂时的过程数据，我会在正式PR提交的时候进行相关的更新（包括删除或者修改部分文档）。

**Review by gaoxh — APPROVED** [review]
+LGTM

**Review by wtt40122 — APPROVED** [review]
+LGTM

**Pull request merged**